packages/driver/src/cypress/log.ts

```diff
@@ -92,17 +92,17 @@ const countLogsByTests = function (tests = {}) {
   }
 
   return _
-  .chain(tests)
-  .flatMap((test) => {
+  .chain(tests) // Creates a lodash wrapper instance that wraps value with explicit method chain sequences enabled. The result of such sequences must be unwrapped with _#value.
```

Comment:
What pain did you have to go through to understand all this and feel the need to document it 😂 . On a serious note, thank you for adding this!

Reply:
Updated